plugins/schedule/base/lib/api/filters/KalturaScheduleEventFilter.php

```diff
@@ -19,6 +19,8 @@ class KalturaScheduleEventFilter extends KalturaScheduleEventBaseFilter
 		"resourceSystemNamesLike" => "_like_resource_system_names",
 		"resourceSystemNamesMultiLikeOr" => "_mlikeor_resource_system_names",
 		"resourceSystemNamesMultiLikeAnd" => "_mlikeand_resource_system_names",
+		"resourceTagsMultiLikeOr" => "_mlikeor_resource_tags",
```

Comment:
@inbal-ben-david because this pertains specifically to the ScheduledEvent and comes from the Resource object is was created with. There's no need to add more bloat.